packages/components/src/components/domainproperties/dataset/DatasetPropertiesPanelFormElements.tsx

```diff
@@ -149,24 +148,43 @@ export class BasicPropertiesFields extends React.PureComponent<BasicPropertiesIn
 interface DataRowUniquenessElementsProps {
     onRadioChange: (evt: any) => any;
     dataRowSetting: number;
+    isFromAssay: boolean;
 }
 
 class DataRowUniquenessElements extends React.PureComponent<DataRowUniquenessElementsProps> {
     render() {
-        const { onRadioChange, dataRowSetting } = this.props;
+        const { onRadioChange, dataRowSetting, isFromAssay } = this.props;
         const radioName = 'dataRowSetting';
 
         return (
             <div className="dataset_data_row_uniqueness_container">
-                <Radio name={radioName} value={0} checked={dataRowSetting == 0} onChange={onRadioChange}>
+                <Radio
+                    name={radioName}
+                    value={0}
+                    checked={dataRowSetting == 0}
+                    onChange={onRadioChange}
+                    disabled={isFromAssay}
+                >
                     Participant ID only (demographic data)
```

Comment:
We will want to update the labels for these radio options to use the study moduleContext subject info.